astropy/tests/command.py

```diff
@@ -177,6 +177,9 @@ def run(self):
 
         # Build a testing install of the package
         self._build_temp_install()
+        # Install a development version of the package so that we have access
+        # to the plugins
+        self.run_command('develop')
```

Comment:
I'm not convinced this is actually necessary, so I may end up removing it before merge. It was originally intended to install plugins that were registered through `setup.py`, but that's no longer happening since I think a better way is just to use a top-level `conftest.py`.